docs/man_pages/screenbuilder/add-dataProvider.md

```diff
@@ -0,0 +1,28 @@
+create
```

Comment:
this is the name of the command and it is used as a title of the page. You should update all your markdowns to have correct title